packages/shared/src/components/modals/PostToSquadModal.tsx

```diff
@@ -57,7 +57,7 @@ function PostToSquadModal({
     if (squadPost) onSharedSuccessfully?.(squadPost);
 
     displayToast(
-      'This post is being processed and will be shared with you Squad shortly',
+      'This post is being processed and will be shared with your Squad shortly',
```

Comment:
@rebelchris I just noticed another thing here, shouldn't we only show this toast when sharing private links, for existing posts there is no need for processing?